server/src/utils/enums/Validations.ts

```diff
@@ -1,4 +1,6 @@
 export enum Validations {
 	PASSWORD_REGEX = '^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,})',
+	EMAIL_REGEX = '^[^@]+@[^@]+$',
+	USERNAME_REGEX = '^[A-Za-z0-9]{3,16}$',
```

Comment:
Same here, i prefer to verify username the same way as front-end
This will be cool to have all validations regex and error messages in a common workspace and have a unique truth source